amd_openvx_extensions/amd_rpp/source/ResizeMirrorNormalizePD.cpp

```diff
@@ -0,0 +1,372 @@
+	STATUS_ERROR_CHECK(vxSetMetaFormatAttribute(metas[3], VX_IMAGE_ATTRIBUTE_WIDTH, &width, sizeof(width)));
+	STATUS_ERROR_CHECK(vxSetMetaFormatAttribute(metas[3], VX_IMAGE_ATTRIBUTE_HEIGHT, &height, sizeof(height)));
+	STATUS_ERROR_CHECK(vxSetMetaFormatAttribute(metas[3], VX_IMAGE_ATTRIBUTE_FORMAT, &df_image, sizeof(df_image)));
+	vxReleaseImage(&input);
+	vxReleaseImage(&output);
+	vxReleaseParameter(&output_param);
+	vxReleaseParameter(&input_param);
+	return status;
+}
+
+static vx_status VX_CALLBACK processResizeMirrorNormalizebatchPD(vx_node node, const vx_reference * parameters, vx_uint32 num)
+{
+	RppStatus status = RPP_SUCCESS;
+	ResizeMirrorNormalizebatchPDLocalData * data = NULL;
+	STATUS_ERROR_CHECK(vxQueryNode(node, VX_NODE_LOCAL_DATA_PTR, &data, sizeof(data)));
+	vx_df_image df_image = VX_DF_IMAGE_VIRT;
+	STATUS_ERROR_CHECK(vxQueryImage((vx_image)parameters[0], VX_IMAGE_ATTRIBUTE_FORMAT, &df_image, sizeof(df_image)));
+
+	if(data->device_type == AGO_TARGET_AFFINITY_GPU) {
+#if ENABLE_OPENCL
+		refreshResizeMirrorNormalizebatchPD(node, parameters, num, data);
+		/* if (df_image == VX_DF_IMAGE_U8 ){
+ 			status = rppi_crop_mirror_normalize_u8_pln1_batchPD_gpu((void *)data->cl_pSrc,data->srcDimensions,data->maxSrcDimensions,(void *)data->cl_pDst,data->dstDimensions,data->maxDstDimensions,data->start_x,data->start_y, data->mean, data->std_dev, data->mirror, data->chnShift ,data->nbatchSize,data->rppHandle);
+		}
+		else if(df_image == VX_DF_IMAGE_RGB) {
+			status = rppi_crop_mirror_normalize_u8_pkd3_batchPD_gpu((void *)data->cl_pSrc,data->srcDimensions,data->maxSrcDimensions,(void *)data->cl_pDst,data->dstDimensions,data->maxDstDimensions,data->start_x,data->start_y, data->mean, data->std_dev, data->mirror, data->chnShift ,data->nbatchSize,data->rppHandle);
+		}
+		return status; */
+#elif ENABLE_HIP
+        refreshResizeMirrorNormalizebatchPD(node, parameters, num, data);
+        if (df_image == VX_DF_IMAGE_U8)
+        {
+            // status = rppi_resize_mirror_normalize_u8_pln1_batchPD_gpu((void *)data->hip_pSrc, data->srcDimensions, data->maxSrcDimensions, (void *)data->hip_pDst, data->dstDimensions, data->maxDstDimensions, data->mean, data->std_dev, data->mirror, data->chnShift, data->nbatchSize, data->rppHandle);
+        }
+        else if (df_image == VX_DF_IMAGE_RGB)
+        {
+            // status = rppi_resize_mirror_normalize_u8_pkd3_batchPD_gpu((void *)data->hip_pSrc, data->srcDimensions, data->maxSrcDimensions, (void *)data->hip_pDst, data->dstDimensions, data->maxDstDimensions, data->mean, data->std_dev, data->mirror, data->chnShift, data->nbatchSize, data->rppHandle);
+			status = rppt_resize_mirror_normalize_gpu((void *)data->hip_pSrc, data->srcDescPtr, (void *)data->hip_pDst, data->dstDescPtr, data->d_dstImgSize, RpptInterpolationType::BILINEAR, data->mean, data->std_dev, data->mirror, data->d_roiTensorPtrSrc, data->roiType, data->rppHandle);
```

Comment:
@sundar If we are internally using rppt call, shouldn't this be like ResizeTensor, Why have we named everywhere as RMNBatchPD?